cli/src/com/redhat/ceylon/common/tools/AbstractTestTool.java

```diff
@@ -92,9 +92,11 @@ public void setVersion(String version) {
         this.version = version;
     }
     
-    @Option(longName = "tap")
-    @Description("Enables the Test Anything Protocol v13.")
-    public void setTap(boolean tap) {
+    @OptionArgument(argumentName = "file")
```

Comment:
Well, if you’re not using the Ant task, then the stdout output is usable to pipe into another program, if you have one that can read from stdin. (Except on the JS backend, where the output will include the `NODE_PATH`.) This wouldn’t really make sense for the HTML report, I think.